src/Content/Widget.php

```diff
@@ -516,6 +516,7 @@ public static function postedByYear(string $url, int $uid, bool $wall)
 			'$cutoff' => $cutoff,
 			'$url' => $url,
 			'$dates' => $ret,
+			'$showless'	=> DI::l10n()->t('show less'),
```

Comment:
This looks like a wrong indentation. At least it is different from the other lines.